### ISSUE TITLE
LibreOffice 24.8

### ISSUE THREAD

**Issue by jeppebundsgaard** [hunspell/hunspell#1023]
I know this is probably not the right place, but this error: https://bugs.documentfoundation.org/show_bug.cgi?id=162514 in LibreOffice is so serious that I wanted to reach out here to hear if it might be a hunspell error anyways, or if someone has the skills to solve it in LibreOffice. 
The bug is making LibreOffice consider all abbreviations with a period in the end incorrect (for example "osv."). This doesn't seem to be a big problem in English, but in Danish and a lot of other languages, I think, this is a huge thing.

**Comment by caolanm** [comment]
yeah, solved in LibreOffice itself via https://bugs.documentfoundation.org/show_bug.cgi?id=162514
